doc/source/whatsnew/v0.19.0.txt

```diff
@@ -749,6 +749,35 @@ New Behavior:
    idx1.difference(idx2)
    idx1.symmetric_difference(idx2)
 
+.. _whatsnew_0190.api.unique_index:
+
+``Index.unique`` consistently returns ``Index``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``Index.unique()`` now return unique values as
```

Comment:
``Index.unique()`` now returns unique values as an ``Index`` of the appropriate ``dtype``